lib/Target/AVR/AVRISelLowering.cpp

```diff
@@ -1295,6 +1295,20 @@ SDValue AVRTargetLowering::LowerCallResult(
 //               Return Value Calling Convention Implementation
 //===----------------------------------------------------------------------===//
 
+bool
+AVRTargetLowering::CanLowerReturn(CallingConv::ID CallConv,
+                                  MachineFunction &MF, bool isVarArg,
+                                  const SmallVectorImpl<ISD::OutputArg> &Outs,
+                                  LLVMContext &Context) const
+{
+  unsigned TotalBits = 0;
+  for (auto Out : Outs) {
+    TotalBits += Out.VT.getSizeInBits();
+  }
+  unsigned MaxBits = 64;
```

Comment:
Add a small comment about why `64` is the max, and that the other data is spilled.